Dashboard/app/js/lib/views/views-public.js

```diff
@@ -52,13 +52,85 @@ Ember.Handlebars.registerHelper('tooltip', function (i18nKey) {
   );
 });
 
+function renderCaddisflyAnswer(json){
+  name = ""
+  imageUrl = ""
+  result = Ember.A();
```

Comment:
`name`, `imageUrl` and `result` without a `var` statement, creates **global** variables